features/mbedtls/platform/inc/shared_rng.h

```diff
@@ -0,0 +1,93 @@
+/*
+ *  shared_rng.h
+ *
+ *  Copyright (C) 2019, Arm Limited, All Rights Reserved
```

Comment:
Should this be 2020? It's a new file never before seen.

Reply:
This is new for public code but internally made last year for a customer

Reply:
OK